website/docs/debug_cable/service_cable.md

```diff
@@ -0,0 +1,54 @@
+---
+id: service-cable
+title: Service Cable Overview
+---
+
+# farm-ng Service Cable
+
+![IMG-9583](https://user-images.githubusercontent.com/64480560/205402624-fa7d8a61-4017-4691-9d24-d6d3ebdeb359.jpg)
+## Overview
+
+Cable used to update and service farm-ng products. It includes rest functionality along with USB communication.
+
+
+![IMG-9585](https://user-images.githubusercontent.com/64480560/205402615-a182d74f-ea46-437b-ad3f-f02f4efbb28d.jpg)
+
+### Product specific uses
+
+#### Dash
+
+- USB interface is for REPL, filesystem access as well as updating firmware and applications,
+
+#### How to connect
+![187535995-d4ad8c7c-05a0-482b-8be9-6b6fb6b87ac3](https://user-images.githubusercontent.com/64480560/206007745-b36c59c7-22dd-4435-9cae-8503956174f3.png)
```

Comment:
add a note to connect to debug port or change text overlay in image to "debug / service"